### PR TITLE
fix for Angleproject : not finding the current beta anymore.

### PR DESCRIPTION
**Pull request by pat357**
My first pull request ever, I hope it works . . .
Patch is simple, but still a bit ugly . . (how to get rid from temp file, without too much extra code ?)
What do you use for aligning ? TAB with off 4 ?
I saw the diff between the master included much more then the actual patch.
I wanted to give the bock under  if [[ $bits = 64. . . . ]] an 1 tab intend, but it  doesn't look that way.

### CODE REVIEW

**Comment by wiiaboo** [comment]
Great, that's a first step. Now:
1. Use the same tab/space indentation as the rest of the file/project (no tabs anywhere here)
2. Don't change code and style in the same commit, makes it confusing to see the changes and might add bugs
3. Use proper commit messages. As a rule, look at how the other commit messages look and do the same

These are general rules applicable to most if not all code repositories, not just this one.

File: build/media-suite_compile.sh [hunk 1]
```diff
@@ -1357,32 +1359,41 @@ if [[ $xpcomp = "n" && $mpv != "n" ]] && pc_exists libavcodec libavformat libsws
...
-        log uninstall make PREFIX="$LOCALDESTDIR" \
-            BINDIR="$LOCALDESTDIR/bin-video" uninstall
-        log configure gyp -Duse_ozone=0 -DOS=win \
+        rm -f __ver.txt
+        curl http://omahaproxy.appspot.com/all?os=win64 -o __ver.txt
+        dos2unix -q __ver.txt
+	betaversion="$(sed -rne '/^win64,(stable|beta)/p' __ver.txt | \
+        	sed -re 's;^win64.*,([^,]+),[^,]+$;\1;' | \
+        	grep -E -e '^[3,4][0-9]+$' - | sort -uV | tail -1)"
+		rm -f __ver.txt
```
Comment (myfreeer) [inline build/media-suite_compile.sh]: Use pipe if you don't want temp file.
```sh
betaversion="$(curl -s https://omahaproxy.appspot.com/all?os=win64 |\
    dos2unix -q | sed -rne '/^win64,(stable|beta)/p' |\
    tr '.' ' ' | awk '{print $3}' | sort -uV | tail -1 )"
```
Also, you can add a fail-safe version after that.
```sh
betaversion="${betaversion:-3112}"
```

**Comment by pat357** [comment]
```
betaversion="$(curl -s https://omahaproxy.appspot.com/all?os=win64 |\
    dos2unix -q | sed -rne '/^win64,(stable|beta)/p' |\
    grep -E -e '^[3,4][0-9]+$' - | sort -uV | tail -1)"
```
How can I properly write code here ?
To undo all the aligning, it might be easier to just start all over again ; how can I do this ?

**Comment by myfreeer** [comment]
@pat357 
> How can I properly write code here ?

Use \`\`\`sh for highlighted code in markdown

> just start all over again 

Edit file in this repo, github will create a new branch for this based on this repo

**Comment by wiiaboo** [comment]
Use a git GUI like [Git Kraken](https://www.gitkraken.com/) to start.

**Comment by wiiaboo** [comment]
@pat357 relevant: https://arstechnica.com/information-technology/2017/06/according-to-statistics-programming-with-spaces-instead-of-tabs-makes-you-rich/

**Comment by wiiaboo** [comment]
Welp, thanks for the PR, but I give up on angle. Tired of dealing with this constantly broken shit.

**Pull request closed without merging**